The dependency target "omptarget.devicertl" of target  "omptarget.rtl.x86_64.unittests" does not exist

I see this error when building 7c3b67d2038cfb.

```
CMake Error at /builddir/build/BUILD/llvm-project-18.0.0.src/openmp/libomptarget/unittests/Plugins/CMakeLists.txt:8 (add_dependencies):
  The dependency target "omptarget.devicertl" of target
  "omptarget.rtl.x86_64.unittests" does not exist.
```

(See https://github.com/kwk/llvm-daily-fedora-rpms/issues/142 or https://download.copr.fedorainfracloud.org/results/@fedora-llvm-team/llvm-snapshots-big-merge-20231222/fedora-38-x86_64/06782394-llvm/builder-live.log.gz)

That means for us the error from https://github.com/llvm/llvm-project/pull/74398 has shifted and while it might be fixed by https://github.com/llvm/llvm-project/pull/76141, we're stuck again.

--- a/openmp/libomptarget/unittests/Plugins/CMakeLists.txt
+++ b/openmp/libomptarget/unittests/Plugins/CMakeLists.txt
@@ -1,4 +1,4 @@
-set(PLUGINS_TEST_COMMON omptarget OMPT omptarget.devicertl)
+set(PLUGINS_TEST_COMMON omptarget)
 set(PLUGINS_TEST_SOURCES NextgenPluginsTest.cpp)
 set(PLUGINS_TEST_INCLUDE ${LIBOMPTARGET_INCLUDE_DIR})
 

